src/crosstab.js

```diff
@@ -377,10 +377,19 @@
         return item;
     }
 
-    function unload() {
+    var hasRestoreListener;
+    function beforeUnload() {
         crosstab.stopKeepalive = true;
+
+        if (!hasRestoreListener) {
+            hasRestoreListener = true;
+            window.addEventListener('DOMContentLoaded', function() {
+                restoreLoop();
+            });
```

Comment:
There's a little more that needs to be done here -- the loop needs to remain active as long as they are still on the page. The current logic will deactivate the loop if they attempt to navigate away at all, but will not restore the loop if the navigation attempt was canceled after `DOMContentLoaded`. This can be fixed by removing the listener from `beforeunload`, and adding it to `unload` once `DOMContentLoaded` fires.